crates/bindings-csharp/Codegen.Tests/fixtures/server/server.csproj

```diff
@@ -2,11 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>net8.0</TargetFramework>
-  </PropertyGroup>
-
-  <PropertyGroup>
-    <IsPackable>false</IsPackable>
-    <DefaultItemExcludes>$(DefaultItemExcludes);snapshots/**/*</DefaultItemExcludes>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
This seems like a bigger change than diagnostics. Does it become the new convention for all C# modules/client libraries too?

Reply:
No, it did the same before, it was just inherited from top-level `Directory.Build.props`. But now that I added a separate `Directory.Build.props` in `fixtures`, I had to manually copy-paste properties that were no longer inherited (`.props` does not inherit from parent directories).